python/lsst/meas/base/forcedPhotCcd.py

```diff
@@ -145,6 +145,19 @@ class ForcedPhotCcdConfig(ForcedPhotImageConfig):
         default=False
     )
 
+    def setDefaults(self):
+        super().setDefaults()
+
+        self.outputSchema.nameTemplate = "forced_src_schema"
+        self.exposure.nameTemplate = "{inputName}"
+        self.exposure.dimensions = ["Instrument", "Visit", "Detector"]
+        self.measCat.nameTemplate = "forced_src"
+        self.measCat.dimensions = ["Instrument", "Visit", "Detector", "SkyMap", "Tract"]
+
+        self.formatTemplateNames({"inputName": "calexp",
```

Comment:
I think this may have gotten out of sync with other changes (or maybe I'm just commenting on a branch you're still working on): I don't see "inputName" in the substitution strings in the config anymore.

Reply:
Oh, nevermind, I see how it works now.  Probably merits a code comment, though.